src/librustc_mir/interpret/validity.rs

```diff
@@ -114,12 +114,11 @@ fn write_path(out: &mut String, path: &Vec<PathElem>) {
             ClosureVar(name) => write!(out, ".<closure-var({})>", name),
             TupleElem(idx) => write!(out, ".{}", idx),
             ArrayElem(idx) => write!(out, "[{}]", idx),
-            Deref =>
-            // This does not match Rust syntax, but it is more readable for long paths -- and
-            // some of the other items here also are not Rust syntax.  Actually we can't
-            // even use the usual syntax because we are just showing the projections,
-            // not the root.
-            {
+            Deref => {
+                // This does not match Rust syntax, but it is more readable for long paths -- and
```

Comment:
Here the separation of pattern and body makes it harder to process and formats a single arm differently from all the above arms. It feels more consistent to have them all be a single line

Reply:
Fair. Note that I didn't make this multi-line, rustfmt did. I did the minimal thing because I feared anything else might just be un-done by rustfmt again (as it did with most of my edits, see the second commit in this PR).